src/App.js

```diff
@@ -1,12 +1,87 @@
-import React from 'react';
-import './App.css';
+
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import PokemonCard from './components/PokemonCard/PokemonCard';
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemonList, setPokemonList] = useState([]);
```

Comment:
Je zal hier wel iets meer in moeten gaan opslaan, zodat je vervolgens nog wel bij de previous en next link kan. Dan raad ik je wel aan om voor een default waarde van `undefined` te gaan
```suggestion
    const [data, setData] = useState();
```